build.gradle

```diff
@@ -19,6 +23,10 @@ buildscript {
 allprojects {
   repositories {
     jcenter()
+    maven {
```

Comment:
You should be able to use the `google()` method here instead of specifying the google maven repo